### PR TITLE
Finetuner integration

### PR DESCRIPTION
**Pull request by atarora**
Added integration of the first fine-tuned model from Jina.
Big thanks to Maximilian Werk and Michael Guenther for the finetuning efforts!

### CODE REVIEW

File: embeddings/app/embeddings.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```
Comment (renekrie) [inline embeddings/app/embeddings.py:1]: Where do we need this import?

**Review by renekrie — CHANGES_REQUESTED** [review]
Thank you @atarora! Please see the comments that I added inline.

File: embeddings/app/embeddings.py (hunk 1 above)
Comment (atarora) [reply embeddings/app/embeddings.py:1]: This is used to enable postponed evaluation of types, not exactly used as a package import but more of a behavior import.

File: docker-compose.yml [hunk 2]
```diff
@@ -233,7 +233,7 @@ services:
     deploy:
       resources:
         limits:
-          memory: 4G
+          memory: 10G
```
[the remark this answers is not in the document]
Comment (atarora) [reply docker-compose.yml:236]: I have tried it with smaller memory but was running into problems, afterall its loading 4 ML models!

File: solr/index-vectors.sh [hunk 3]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 DATA_DIR="./solr/data"
-
```
[the remark this answers is not in the document]
Comment (atarora) [reply solr/index-vectors.sh:4]: This definitely needs a fix, I upload the vector files to s3 and change the script to reflect downloading and indexing finetuned vector data.

**Pull request merged**